packages/web-sdk/src/config/getWebInstrumentations.ts

```diff
@@ -32,5 +32,11 @@ export function getWebInstrumentations(options: GetWebInstrumentationsOptions =
     );
   }
 
+  if (options.captureWebVitalsAttribution === true) {
+    instrumentations.push(new WebVitalsWithAttributionInstrumentation());
+  } else {
+    instrumentations.push(new WebVitalsInstrumentation());
+  }
```

Comment:
this way the version of 'web-vitals/attribution' will always be present in the bundle even if the client uses the lighter version

This is because the import of both instruments are carried out in this module.

I can't do a lazy import here because the function is sync. What is the best way to get around this problem? Removing the `captureWebVitalsAttribution` option is also an option